conda/recipes/pyraft/conda_build_config.yaml

```diff
@@ -11,7 +11,7 @@ sysroot_version:
   - "2.17"
 
 ucx_version:
-  - "1.12.1"
+  - "1.13.0"
```

Comment:
Do you mind bumping the versions in the conda environment files also? Otherwise, it looks good to me.

Reply:
Sorry, I missed those. Done now in https://github.com/rapidsai/raft/pull/736/commits/6b8dc21c481d4f5a0871cacf2934f4891f2311ce .